xray/commands/audit/sca/java/mvn.go

```diff
@@ -1,98 +1,175 @@
 package java
 
 import (
+	_ "embed"
+	"encoding/base64"
 	"errors"
 	"fmt"
-	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
-	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
-	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
-	mvnutils "github.com/jfrog/jfrog-cli-core/v2/utils/mvn"
-	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
 	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
 	"github.com/jfrog/jfrog-client-go/utils/log"
 	xrayUtils "github.com/jfrog/jfrog-client-go/xray/services/utils"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
 )
 
-func buildMvnDependencyTree(params *DependencyTreeParams) (modules []*xrayUtils.GraphNode, uniqueDeps []string, err error) {
-	buildConfiguration, cleanBuild := createBuildConfiguration("audit-mvn")
-	defer func() {
-		err = errors.Join(err, cleanBuild())
-	}()
+const (
+	mavenDepTreeJarFile    = "maven-dep-tree.jar"
+	mavenDepTreeOutputFile = "mavendeptree.out"
+	mavenDepTreeVersion    = "1.0.2"
+	TreeCmd                = "tree"
+	ProjectsCmd            = "projects"
+	settingsXmlFile        = "settings.xml"
+)
 
-	mvnProps := CreateMvnProps(params.DepsRepo, params.Server)
-	err = runMvn(buildConfiguration, params.InsecureTls, params.IgnoreConfigFile, params.UseWrapper, mvnProps)
+//go:embed resources/settings.xml
+var settingsXmlTemplate string
+
+//go:embed resources/maven-dep-tree.jar
+var mavenDepTreeJar []byte
+
+type MavenDepTreeManager struct {
+	*DepTreeManager
```

Comment:
Any reason to use a pointer?
```suggestion
	DepTreeManager
```